tests/unit/test_termio.py

```diff
@@ -30,3 +36,20 @@ def test_term_interactive(ps1, force_flag, expect, monkeypatch):
         monkeypatch.delitem(os.environ, "GLOBUS_CLI_INTERACTIVE", raising=False)
 
     assert term_is_interactive() == expect
+
+
+def test_format_record_list(capsys):
+    data = [
+        {"bird": "Killdeer", "wingspan": 46},
+        {"bird": "Franklin's Gull", "wingspan": 91},
+    ]
```

Comment:
I'm disappointed that you chose not to include the scientific names, flavor text, or ability text. 😜

Reply:
You know what they say about programmers and laziness...and it seems like the reference still got across :wink:

Reply:
I wasn't sure until I checked that those numbers are the wingspans on the cards. I'm glad to have it in our test data! Look out for Tardigrades and Deimos Down in my PRs.